sys/kern/vfs_syscalls.c

```diff
@@ -106,6 +107,23 @@ static int vfs_open(file_t *f, char *pathname, int flags, int mode) {
   return error;
 }
 
+/* Unlocks and decrements reference counter for both vnode and its parent */
+static void vnode_put2(vnode_t *v, vnode_t *dv) {
```

Comment:
Maybe `vnode_put_both` and `vnode_drop_both`?

Reply:
Done.